.travis.yml

```diff
@@ -14,16 +15,18 @@ env:
 
 matrix:
   include:
-    - { python: 2.7, env: S3=0 TOXENV=py27-dj111 }
-    - { python: 2.7, env: S3=1 TOXENV=py27-dj111 }
-    - { python: 2.7, env: S3=0 TOXENV=py27-dj111-grp }
-    - { python: 3.6, env: S3=0 TOXENV=py36-dj111 }
-    - { python: 3.6, env: S3=0 TOXENV=py36-dj20 }
-    - { python: 3.6, env: S3=1 TOXENV=py36-dj20 }
-    - { python: 3.6, env: S3=0 TOXENV=py36-dj20-grp }
-    - { python: 3.6, env: TOXENV=py36-dj21 }
+    - { python: 2.7, env: S3=0 TOXENV=py27-dj111-pil27 }
+    - { python: 2.7, env: S3=0 TOXENV=py27-dj111-pil54 }
+    - { python: 2.7, env: S3=1 TOXENV=py27-dj111-pil54 }
+    - { python: 2.7, env: S3=0 TOXENV=py27-dj111-grp-pil54 }
+    - { python: 3.6, env: S3=0 TOXENV=py36-dj111-pil54 }
+    - { python: 3.6, env: S3=0 TOXENV=py36-dj20-pil54 }
+    - { python: 3.6, env: S3=1 TOXENV=py36-dj20-pil54 }
+    - { python: 3.6, env: S3=0 TOXENV=py36-dj20-grp-pil54 }
+    - { python: 3.6, env: S3=0 TOXENV=py36-dj21-pil54 }
   allow_failures:
-    - env: TOXENV=py36-dj21
+    - env: S3=0 TOXENV=py27-dj111-pil27
```

Comment:
Is this supposed to be in `allow_failures`? It appears to pass in travis.

Reply:
It is, but travis seems to installing `pillow==6.2.0` instead of `pillow==2.7.0` for that test run (see [me](https://travis-ci.org/github/theatlantic/django-cropduster/jobs/668758315#L403)). I suspect because of the [update to setup.py](https://github.com/theatlantic/django-cropduster/pull/61/files#diff-2eeaed663bd0d25b7e608891384b7298R24), but I would've thought tox/travis would ignore 🤔